codespell_lib/data/dictionary.txt

```diff
@@ -23928,6 +23934,7 @@ seperaors->separators
 seperare->separate
 seperared->separated
 seperares->separates
+seperat->separate
```

Comment:
Can you add the -d and -ly versions of this

Reply:
they are already in the dict.

Reply:
As in seperatd and seperatly which aren't present.

Reply:
If I'm not totally stupid, they are present. :-)

Reply:
I'm not sure how I missed them before, but expanding the code folding just now, Github failed to render a huge chunk so maybe it was that again.